<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>828</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>828</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>T1Modbus_Data_Collector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <item row="2" column="4">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>DB Status :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="1" colspan="5">
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string>192.168.172.210</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QLineEdit" name="lineEdit_11">
           <property name="text">
            <string>SELECT count(*) FROM TB_CHECK_ENVIRON_IOT</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1" colspan="3">
          <widget class="QComboBox" name="comboBox">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Modbus/TCP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Modbus/RTU</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>COM Port :</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0" colspan="6">
          <widget class="QLabel" name="label_28">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Threshold Value Setting</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="lineEdit_7">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>1521</string>
           </property>
          </widget>
         </item>
         <item row="14" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Registers No. :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_14">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Service</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4" colspan="2">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Query Test</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Start Address :</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="5">
          <widget class="QListWidget" name="listWidget_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>User</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="6">
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Modbus Protocol Communication for Logging </string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>SQL Query</string>
           </property>
          </widget>
         </item>
         <item row="14" column="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Time Interval(ms) :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Disconnected</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="3">
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>접속 상태: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_9">
           <property name="text">
            <string>ssc</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_6">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>192.168.32.10</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLineEdit" name="lineEdit_8">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>orcl</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="lineEdit_10">
           <property name="text">
            <string>pop</string>
           </property>
          </widget>
         </item>
         <item row="17" column="4">
          <widget class="QLabel" name="label_27">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>Min</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="19" column="2">
          <widget class="QLineEdit" name="lineEdit_12">
           <property name="text">
            <string>30</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="19" column="5">
          <widget class="QLineEdit" name="lineEdit_13">
           <property name="text">
            <string>30</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item row="19" column="4">
          <widget class="QLineEdit" name="lineEdit_14">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="17" column="2">
          <widget class="QLabel" name="label_22">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>Max</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="14" column="5">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="text">
            <string>10000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="11" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="text">
            <string>COM1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="19" column="1">
          <widget class="QLineEdit" name="lineEdit_15">
           <property name="text">
            <string>5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>IP Address :</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>SQL Response</string>
           </property>
          </widget>
         </item>
         <item row="11" column="4" colspan="2">
          <widget class="QComboBox" name="comboBox_2">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="currentText">
            <string>9600</string>
           </property>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="5">
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
         <item row="8" column="4">
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item row="9" column="3" colspan="3">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Disconnected</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Mode Selection</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QLabel" name="label_26">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>Min</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="11" column="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Baud Rate :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="6">
          <widget class="QLabel" name="label_17">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Oracle DB Connection for Test and Setting</string>
           </property>
          </widget>
         </item>
         <item row="17" column="5">
          <widget class="QLabel" name="label_23">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>Max</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="14" column="3">
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="text">
            <string>6</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="20" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>Node 2 :</string>
           </property>
          </widget>
         </item>
         <item row="19" column="0">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>Node 1 :</string>
           </property>
          </widget>
         </item>
         <item row="16" column="1" colspan="2">
          <widget class="QLabel" name="label_24">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>Tempeature</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="16" column="4" colspan="2">
          <widget class="QLabel" name="label_25">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>Humidity</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="21" column="0">
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>Node 3 :</string>
           </property>
          </widget>
         </item>
         <item row="20" column="1">
          <widget class="QLineEdit" name="lineEdit_16">
           <property name="text">
            <string>5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="20" column="2">
          <widget class="QLineEdit" name="lineEdit_17">
           <property name="text">
            <string>30</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="20" column="4">
          <widget class="QLineEdit" name="lineEdit_18">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="20" column="5">
          <widget class="QLineEdit" name="lineEdit_19">
           <property name="text">
            <string>30</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="21" column="1">
          <widget class="QLineEdit" name="lineEdit_20">
           <property name="text">
            <string>5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="21" column="2">
          <widget class="QLineEdit" name="lineEdit_21">
           <property name="text">
            <string>30</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="21" column="4">
          <widget class="QLineEdit" name="lineEdit_22">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="21" column="5">
          <widget class="QLineEdit" name="lineEdit_23">
           <property name="text">
            <string>30</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="19" column="3">
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>Node 1 :</string>
           </property>
          </widget>
         </item>
         <item row="20" column="3">
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>Node 2 :</string>
           </property>
          </widget>
         </item>
         <item row="21" column="3">
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>Node 3:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <item>
      <property name="text">
       <string>Node1</string>
      </property>
      <property name="background">
       <brush brushstyle="Dense4Pattern">
        <color alpha="50">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>200</blue>
        </color>
       </brush>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Node2</string>
      </property>
      <property name="background">
       <brush brushstyle="Dense4Pattern">
        <color alpha="50">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>200</blue>
        </color>
       </brush>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Node3</string>
      </property>
      <property name="background">
       <brush brushstyle="Dense4Pattern">
        <color alpha="50">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>200</blue>
        </color>
       </brush>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Updated Time :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>slot_4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot_5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_1()</slot>
  <slot>slot_2()</slot>
  <slot>slot_3()</slot>
  <slot>slot_4()</slot>
  <slot>slot_5()</slot>
  <slot>slot_6()</slot>
 </slots>
</ui>
